document/chapters/01-introduction.tex

```diff
@@ -9,11 +9,12 @@
 
 \chapter{Introduction}\label{ch:introduction}
 
-Methods for modeling radiation transport to determine particle flux, or derived
-quantities, across space, angle, energy and time known as \textit{phase
-  space}. The behavior of these particles is described by the linear Boltzmann
-transport equation \cite{Ulam_1949}. Deterministic codes solve this transport
-equation by discretizing the phase space of the problem domain, but time and memory
+This work focuses on methods for modeling radiation transport to determine
+particle flux, or derived quantities, across the dimensions of space, angle,
+energy and time known as \textit{phase space}. The behavior of these particles
+is described by the linear Boltzmann transport equation
+\cite{Ulam_1949}. Deterministic codes solve this transport equation by
+discretizing the phase space of the problem domain, but time and memory
```

Comment:
Pro-tip: avoid changing line breaks, even if it means some strangely short lines, to facilitate review by minimizing diffs....